index.js

```diff
@@ -2487,6 +2487,20 @@
   }
   S.rights = def('rights', {}, [$.Array($Either(a, b)), $.Array(b)], rights);
 
+  //# tagBy :: (a -> Boolean) -> a -> Either a a
+  //.
+  //. Takes a predicate and a value and returns a Right of the value if it
+  //. satisfies the predicate and a Left of the value otherwise.
```

Comment:
I'd like to see a comma after the first occurrence of _value_.

I think a semicolon would be more appropriate than _and_ after the second occurrence of _predicate_.